Release notes for Red Hat Developer Hub 1.4- Update Release Note Texts (#758)
redhat-developer/red-hat-developers-documentation-rhdh: modules/release-notes/snip-bug-fix-rhidp-4240.adoc

[id="bug-fix-rhidp-4240"]

> = Removed the pre-configured custom resources from the k8s configuration

Reviewer: ```suggestion
= Removed the pre-configured custom resources from the Kubernetes configuration
```

> Before this update, the custom resources in k8s configuration were pre-configured. As a result, users could see tekton warnings without configuring the custom resources in k8s.

Reviewer: ```suggestion
Before this update, the custom resources in Kubernetes configuration were pre-configured. As a result, users could see Tekton warnings without configuring the custom resources in Kubernetes.
```

> With this update, the pre-configured custom resources from the k8s configuration are removed. Therefore, users can custom resources to the Kubernetes configuration based on their requirements, preventing unrelated warnings from appearing.

Reviewer: ```suggestion
This update removes the pre-configured custom resources from the Kubernetes configuration. Therefore, users can customize resources to the Kubernetes configuration based on their requirements, preventing unrelated warnings from appearing.
```
